sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultStep.java

```diff
@@ -143,16 +140,6 @@ public Status getStatus() {
         }
     }
 
-    @Override
-    public List<Element<?>> getChildren() {
-        return Collections.emptyList();
-    }
-
-    @Override
```

Comment:
Steps cannot be interrupted in the middle of performing work. Interrupt takes effect at the phase level, after the current step has completed. This has always effectively been the case but with this change it's more explicit.